robot/Loadtest.robot

```diff
@@ -19,11 +20,21 @@ Perform load test
     Set eUPF QER    qer_id=${0}
 
     ${PACKET}  Create GTP-U Packet
-    #${handle}=     Run Method Async    Sendpfast  ${PACKET}  iface=lo  file_cache=true  loop=${0}  replay_args=@{TCPREPLAY_EXTRA_ARGS}  parse_results=true
-    #${return_value}=     Wait Async All     timeout=5
-    #${RESULT}    get async return    handle=${handle}     timeout=${15}
-    @{TCPREPLAY_EXTRA_ARGS}   Create List    --limit=${TCPREPLAY_LIMIT}
-    ${RESULT}    Sendpfast  ${PACKET}  iface=lo  file_cache=true  loop=${0}  replay_args=@{TCPREPLAY_EXTRA_ARGS}  parse_results=true
+    ${TCPREPLAY_EXTRA_ARGS}   Create List    --limit=${TCPREPLAY_LIMIT}
+    FOR    ${i}    IN RANGE    ${TCPREPLAY_THREADS}
+           Exit For Loop If    ${i} == ${TCPREPLAY_THREADS}
+           ${handle1}=     Run Keyword Async    Sendpfast  ${PACKET}  ${None}  ${None}  ${False}  ${0}  ${True}  lo  ${TCPREPLAY_EXTRA_ARGS}  ${True}
```

Comment:
Is it possible to use named arguments here?

Reply:
Well, runKeywordAsync library says no ("Note: It takes only args as arguments, kwargs is not supported"), since it is using [run_keyword(self, name, *args)](https://github.com/robotframework/robotframework/blob/d161a15774dd4149fe7d01bb567e2687a4cc681e/src/robot/libraries/BuiltIn.py#L1839) function from BuiltIn robot framework library.  

However, yes, I suppose, it is possible to bypass this. For example, write our own specific method (combination of Run Method Async and Sendpfast). Or try to call sendpfast as a method from scapy library, not as a keyword, which will allow us to use [Run Method Async](https://github.com/awadhn/robotframework-run-keyword-async/blob/master/runKeywordAsync/runKeywordAsync.py#L13), which has **kwargs.

Reply:
Haven't figured it out yet. Later?